frontend/src/metabase/entities/users.js

```diff
@@ -35,8 +35,14 @@ const Users = createEntity({
   path: "/api/user",
 
   api: {
-    list: ({ recipients = false, ...args }) =>
-      recipients ? getRecipientsList() : getUserList(args),
+    list: ({ recipients = false, ...args }, dispatch) =>
```

Comment:
We should override all 5 default endpoints (i.e. also `get`, `update`, `delete`).
For the ones that are unused we should `throw` to prevent adding new code using legacy stuff.

Reply:
Done.
Thanks

Reply:
@nemanjaglumac please don't forget about `delete` :pray: 

It should use `deactivateUser` endpoint.